src/pages/invoices/common/hooks/useResolveTranslation.ts

```diff
@@ -12,10 +12,13 @@ import { useCurrentCompany } from '$app/common/hooks/useCurrentCompany';
 import { resolveKey } from '$app/pages/invoices/common/helpers/resolve-key';
 import { useTranslation } from 'react-i18next';
 
-export function useResolveTranslation() {
+export function useResolveTranslation({ type }: { type?: 'product' | 'task' }) {
```

Comment:
If this grows, extract to interface, please.

Reply:
Extracted